src/main/java/org/candlepin/subscriptions/ApplicationConfiguration.java

```diff
@@ -186,4 +194,9 @@ public RdsSslBeanPostProcessor rdsSslBeanPostProcessor(Environment env) {
   public CertInfoContributor certInfoContributor(ApplicationContext context) {
     return new CertInfoContributor(context);
   }
+
+  @Bean
+  public ConsoleCloudEventParser cloudEventParser() {
+    return new ConsoleCloudEventParser();
```

Comment:
I've just found that this constructor accepts an objectMapper instance, so I think we should wire the Spring objectMapper instance here.